.tx/config

```diff
@@ -0,0 +1,9 @@
+[main]
+host = https://app.transifex.com
+
+[o:foreman:p:foreman:r:hammer_cli_foreman_scc_manager]
+file_filter = locale/<lang>/hammer_cli_foreman_scc_manager.edit.po
+source_file = locale/hammer_cli_foreman_scc_manager.pot
+source_lang = en
+type = PO
+resource_name = hammer_cli_foreman_scc_manager
```

Comment:
Do you actually submit that config file? I haven't seen it in any other projects.

Reply:
My bad, I did not search well enough :)